:memo: Rework activity reference (#2766)
platformsh/platformsh-docs: docs/src/integrations/activity/_index.md

```

That installs and enables the `my_script.js` file as an activity script on the current project.

Don't run the `integration:add` command a second time,
or it installs a second integration with the same code.

## Updating

To update an existing activity script, follow these steps:

1. Get the activity script's ID by running the following command:

   ```bash
   platform integrations
   ```

   This returns something like the following:

   ```bash
   +---------------+--------------+--------------+
   | ID            | Type         | Summary      |
   +---------------+--------------+--------------+
   | nadbowmhd67do | script       | ...          |
   | rcqf6b69jdcx6 | health.email | From:        |
   |               |              | To: #admins  |
   +---------------+--------------+--------------+
   ```

2. Update the integration by running the following command:

   ```bash
   platform integration:update --file ./my_script.js {{<variable "SCRIPT_ID" >}}
   ```

   That updates the integration in place, permanently overwriting the previous version.

> 3. Test the activity script update, by triggering a redeployment with the following command:

Reviewer: ```suggestion
3. Test the activity script update by triggering a redeployment with the following command:
```

   ```bash
   platform redeploy
   ```

## Removing

To disable an activity script, follow these steps:

1. Get the activity script's ID by running the following command:

   ```bash
   platform integrations
   ```

   This returns something like the following:

   ```bash
   +---------------+--------------+--------------+
   | ID            | Type         | Summary      |
   +---------------+--------------+--------------+
   | nadbowmhd67do | script       | ...          |
   | rcqf6b69jdcx6 | health.email | From:        |
   |               |              | To: #admins  |
   +---------------+--------------+--------------+
   ```

2. Delete the integration by running the following command:

   ```bash
   platform integration:delete {{<variable "SCRIPT_ID" >}}
   ```

## Debugging

Get activity logs by running the following command:

```bash
platform integration:activities
```

Every time your activity script runs it generates a new log entry, including the output from the script.

> Any output produced by `console.log` is available in the activity log, and that's the recommended way to debug scripts.

Reviewer: ```suggestion
Any output produced by `console.log` is available in the activity log, which is the recommended way to debug scripts.
```

## Available APIs

Activity scripts can be written in ES2021 and don't support installing additional packages.

> We provide a series of [utility functions you can reuse](./utility.md)

Reviewer: :hammer_and_wrench: 
```suggestion
There are a series of [utility functions you can reuse](./utility.md)
```
